pages/treeherder.py

```diff
@@ -152,6 +207,22 @@ def open_perfherder_page(self):
     def open_repos_menu(self):
         self.find_element(*self._repos_menu_locator).click()
 
+    def pin_job_and_enter_bug_number(self, bug_number):
```

Comment:
We can pin and enter bug number, but it will only display on a website.
Should I remove the test and related code?